draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -409,7 +407,7 @@ shown in {{ref-option-list}}.
 
 A DCCP endpoint negotiates the Multipath Capable Feature to determine whether multipath extensions can be enabled for a DCCP connection.
 
-The Multipath Capable feature (MP_CAPABLE) has feature number 10 and follows the structure for features given in {{RFC4340}} Section 6. Beside the negotiation of the feature itself, also one or several values can be exchanged. The value field specified here for the Multipath Capable feature has a length of one-byte and can be repeated several times within the DCCP option for feature negotiation if required for example to announce support of different versions of the protocol. For that, the leftmost four bits in {{ref-mp-capable-format}} specify the compatible version of the
+The Multipath Capable feature (MP_CAPABLE) has feature number 10 and follows the structure for features given in {{RFC4340}} Section 6. Beside the negotiation of the feature itself, also one or several values can be exchanged. The value field specified here for the Multipath Capable feature has a length of one-byte and can be repeated several times within the DCCP option for feature negotiation if required for example to announce support of different versions of ol. For that, the leftmost four bits in {{ref-mp-capable-format}} specify the compatible version of the
```

Comment:
What is ths ol?

Reply:
Incorrect substitution reverted in https://github.com/tsvwg/ietf-multipath-dccp/pull/309/commits/327a8b7e554d35cbec5fe1a1ce34e0159362d883 and https://github.com/tsvwg/ietf-multipath-dccp/pull/309/commits/4cc65e8ccb3f2b8aee9a86006e4ceff7e47691ed.